examples/credential_metadata_jwt_vc_json.json

```diff
@@ -0,0 +1,52 @@
+{
+    "format": "jwt_vc_json",
+    "id": "UniversityDegree_JWT",
+    "type": [
+        "VerifiableCredential",
+        "UniversityDegreeCredential"
+    ],
+    "cryptographic_binding_methods_supported": [
+        "did:example"
+    ],
+    "cryptographic_suites_supported": [
+        "ES256K"
+    ],
+    "display": [
+        {
+            "name": "University Credential",
+            "locale": "en-US",
+            "logo": {
+                "url": "https://exampleuniversity.com/public/logo.png",
```

Comment:
we should mandate the SVG file format for the logo, since it grants a high quality rendering in any kind of display resolution

Reply:
can you please open a separate issue on that so that we can discuss it there? thanks

Reply:
Here https://github.com/vcstuff/oid4vc-haip-sd-jwt-vc/issues/58

Reply:
thanks